pyspecdata/core.py

```diff
@@ -6918,20 +6932,25 @@ def __init__(self,*args,**kwargs):
         return
     @property
     def function_string(self):
-        r'''A property of the fitdata class which stores a string output of the functional form of the desired fit expression provided in func:`functional_form` in LaTeX format'''
+        r'''A property of the fitdata class which stores a string
+        output of the functional form of the desired fit expression
+        provided in func:`functional_form` in LaTeX format'''
         retval = sympy.latex(self.symbolic_expr).replace('$','')
         return r'$f(%s)='%(sympy.latex(self.fit_axis)) + retval + r'$'
     @function_string.setter
     def function_string(self):
         raise ValueError("You cannot set the string directly -- change the functional_form property instead!")
     @property
     def functional_form(self):
-        r'''A property of the fitdata class which is set by the user, takes as input a sympy expression of the desired fit expression'''
+        r'''A property of the fitdata class which is set by the user,
+        takes as input a sympy expression of the desired fit
+        expression'''
```

Comment:
I would give a basic example here, or in the fitdata docstring (if the latter, you can refer to it here as ``class:`fitdata```)